requirements/requirements.in

```diff
@@ -45,7 +45,6 @@ simplejson~=3.11
 sh==1.09
 gevent==1.2.2
 greenlet==0.4.12
-numpy==1.7.1
```

Comment:
I think you'll also want to add this to uninstall-requirements.txt. There's also a reference to it in PRE_DOCKER_SETUP

Reply:
fun fact that `numpy` is actually our largest dependency:

![image](https://user-images.githubusercontent.com/1471773/53063960-1003cf00-3494-11e9-89bd-a489d21d7f7d.png)